packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -1,6 +1,7 @@
 .edit-post-visual-editor {
 	position: relative;
-
+	display: flex;
+	flex-flow: column;
```

Comment:
It doesn't seem to cause any issues, but I couldn't quite see why this change was made.

Reply:
`flex-flow: column;` is a magical way to stack elements with the total height remaining 100%. If one of the elements sets its height to 100%, it will just fill up the remaining height. Cc @jasmussen

Reply:
Very cool indeed. 

I'll keep an eye on any flex related issues that might occur from this, but in my testing this worked as intended, and it lets us avoid doing something like `height: calc(100% - #{ $block-toolbar-height } )` for the canvas.